src/m365/entra/commands/app/app-permission-add.ts

```diff
@@ -137,13 +143,45 @@ class EntraAppPermissionAddCommand extends GraphCommand {
     }
   }
 
+  private async getAppObjectId(appName: string): Promise<string> {
```

Comment:
In the current setup, this function contains an "extra" http call. We should just execute one HTTP call to get the app object, whatever option has been used.